website/docs/community/contributors.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Contributors
+sidebar_position: 2
+---
+
+# Contributors
+
+## Mihai Carabaș
+
+<img src="/img/people/mihai_carabas.png" width="200" />
+
+Hi!
+I'm Mihai, I am part of the OpenEdu project (21-COP-0016).
+
+I am contributing on the infrastructure architecture and I supervise the infrastructure deployment project activities.
+I am also engaged in content reviewing the Operating Systems class.
+
+I am a Professor at University POLITEHNICA of Bucharest, Romania, the Computer Science and Engineering Department.
+I am primarily interested in operating systems, computer architectures, cluster, cloud and grid computing.
+
+## Răzvan Rughiniș
+
+<img src="/img/people/razvan_rughinis.jpg" width="200" />
+
+Hi!
+I am Răzvan, I am a contributor to the OpenEdu Project (21-COP-0016).
+
+I am a Professor at the Faculty of Automatic Control and Computers, University POLITEHNICA of Bucharest.
+My research, teaching, and doctoral supervision focus on cybersecurity, privacy, and data science.
+
+I am co-founder and coordinator of the [Innovation Labs acceleration program](https://www.innovationlabs.ro/) for young tech teams, since 2013, bringing together the Romanian academia and ICT business community to energize a vibrant startup ecosystem.
```

Comment:
```suggestion
Since 2013 I am the co-founder and coordinator of the [Innovation Labs acceleration program](https://www.innovationlabs.ro/) for young tech teams, bringing together the Romanian academia and ICT business community to energize a vibrant startup ecosystem.
```